apache/incubator-metron
# METRON-395 Fix Metron Bro parser not parsing some timestamp values

This PR addresses https://issues.apache.org/jira/browse/METRON-395

This fix corrects the BasicBroParser so that it will properly handle Double timestamp values that have less than 3 digits, e.g. 123.0, 123.1, 123.11. In the previous examples, the current code would throw an index out of bounds exception.

Verified in full-dev-platform.

**Testing:**

- Deploy the bro parser topology.
- Add a record to dns.log or http.log with a ts value that has less than 3 significant decimal places.
- Confirm the record(s) make it into indexing.
